pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -116,8 +126,8 @@ let
   # doesn't work on it. For now it is fine since ltoSupport implies no Darwin.
   buildStdenv = if ltoSupport
                 # LTO requires LLVM bintools including ld.lld and llvm-ar.
-                then overrideCC llvmPackages.stdenv (llvmPackages.stdenv.cc.override {
-                  inherit (llvmPackages) bintools;
+                then overrideCC llvmPackagesStdenv.stdenv (llvmPackagesStdenv.stdenv.cc.override {
```

Comment:
Is stdenv.stdenv needed?

Reply:
llvmPackagesStdenv is actually a name for llvmPackages *for* Stdenv, not a stdenv